src/components/dashboard/main-content/course-enrollments/course-cards/mark-complete-modal/ModalBody.jsx

```diff
@@ -13,16 +13,16 @@ const ModalBody = () => {
     <>
       {confirmError && <ModalError />}
       <p className="m-0">
-        Are you sure you want to archive
+        Are you sure you want to save
         {' '}
-        <a href={courseLink}>{courseTitle}</a>?
+        <a href={courseLink}>{courseTitle}</a>
         {' '}
-        You will remain enrolled, but the course will
+        for later? You will remain enrolled, but the course will
         no longer appear as &quot;In Progress&quot;.
       </p>
       <p className="mt-2">
         As long as your license is valid, you can resume the course by clicking
-        &quot;Continue Learning&quot; under your list of archived courses.
+        &quot;Continue Learning&quot; under your list of courses saved for later.
```

Comment:
change to 'Move to In Progress'

Reply:
Got it, yeah, I had it as that initially, but there was some text in the modal that used "Continue learning". Which, tbh, I kind of like. Should I change the modal text as well?

Reply:
yup I think the only place we will now use `continue learning`, is on the buttons that are already there i the cards ... but all text associated with this workflow will use Move To In Progress, and Save for later